docs/core/diagnostics/includes/available-counters.md

```diff
@@ -62,12 +62,12 @@ The following counters are published as part of the [ASP.NET Core Kestrel web se
 | Counter | Description |
 |--|--|
 | :::no-loc text="Connection Queue Length"::: (`connection-queue-length`) | The current length of the connection queue |
-| :::no-loc text="Connection Rate"::: (`connections-per-second`) | The number of connections per second to the web server |
+| :::no-loc text="Connection Rate"::: (`connections-per-second`) | The number of connections per update interval to the web server |
 | :::no-loc text="Current Connections"::: (`current-connections`) | The current number of active connections to the web server |
 | :::no-loc text="Current TLS Handshakes"::: (`current-tls-handshakes`) | The current number of TLS handshakes |
 | :::no-loc text="Current Upgraded Requests (WebSockets)"::: (`current-upgraded-requests`) | The current number of upgraded requests (WebSockets) |
 | :::no-loc text="Failed TLS Handshakes"::: (`failed-tls-handshakes`) | The total number of failed TLS handshakes |
 | :::no-loc text="Request Queue Length"::: (`request-queue-length`) | The current length of the request queue |
-| :::no-loc text="TLS Handshake Rate"::: (`tls-handshakes-per-second`) | The number of TLS handshakes per second |
+| :::no-loc text="TLS Handshake Rate"::: (`tls-handshakes-per-second`) | The number of TLS handshakes per update interval |
```

Comment:
The name `tls-handshakes-per-second` is now out of sync with the description

Reply:
Same as above.